packages/terra-base/src/Base.jsx

```diff
@@ -1,38 +1,74 @@
 import React, { PropTypes } from 'react';
-import './Base.scss';
+
+import { I18nProvider, i18nLoader } from 'terra-i18n';
+
+import BaseStyles from './BaseStyles';
 
 const propTypes = {
   /**
    * The component(s) that will be wrapped by `<Base />`.
    */
   children: PropTypes.node.isRequired,
+  /**
+   * Customized translations provided by consuming application.
+   */
+  customMessages: PropTypes.object,
```

Comment:
Yes. This is only for the single locale. `messages` in i18n is also only for single locale. I will update the comments to make it clear.